configure.ac

```diff
@@ -85,7 +85,8 @@ AC_PATH_TOOL(STRIP, strip)
 AC_PATH_TOOL(GCOV, gcov)
 AC_PATH_PROG(LCOV, lcov)
 dnl Python 3.x is supported from 3.4 on (see https://github.com/bitcoin/bitcoin/issues/7893)
```

Comment:
I'd prefer not to link to github issues unless necessary to keep the source code "self contained" as possible.

I guess you could delete this comment altogether, since we added `doc/dependencies.md` in the meantime.